scripts/index.js

```diff
@@ -19,6 +19,19 @@
     console.warn('unhandled input type ' + el.type);
   };
 
+  var newEvent = function(name) {
+    var event
+
+    if (typeof window.Event === 'function') {
+      event = new window.Event('submit');
```

Comment:
I think this should be `name` not `'submit'`

Reply:
yep